main.py

```diff
@@ -0,0 +1,8 @@
+import numpy as np
+from patblib import Path
+
+
+def z(x, y):
+    x = np.array(x)
```

Comment:
Assigning x to a numpy array can happen in the pipeline outside of this function. We want the function to be simple only do what it says it's going to do

Reply:
I set the type of x to already be a numpy array. This line was removed here 45cd48dd64bae55521182d52a6cf225c0d39076b